### PR TITLE
Make HipChat integration enterprise friendly

### LINKED ISSUE #160: Allow URL for HipChat Integration
Currently the HipChat Integration is hard-coded, this should be an optional string to allow on-prem hipchat integrations to work as well.

### PR DESCRIPTION
**Pull request by discorick**
The api endpoint is now configurable for HipChat

Fixes #160

### CODE REVIEW

File: ember-app/app/templates/settings/integrations/hipchat.hbs [hunk 1]
```diff
@@ -9,6 +9,10 @@
     <label>
       {{input value=model.authToken placeholder="Authorization Token" required=true }}
     </label>
+    <p> HipChat Endpoint (no trailing slashes): </p>
+    <label>
+      {{input value=model.endpoint required=true }}
```
Comment (drusellers) [inline ember-app/app/templates/settings/integrations/hipchat.hbs]: Is it worth having some example values?

Comment (discorick) [inline ember-app/app/templates/settings/integrations/hipchat.hbs]: The value is pre filled as the default HipChat API endpoint `"https://api.hipchat.com"` as per `ember-app/app/controllers/settings/integrations.js` Line 8, are you suggesting even more examples ?

**Comment by dahlbyk** [comment]
Change looks good. Do we care to drop a trailing slash if the user happens to specify one (resulting in `https://api.hipchat.com//v2/room/foo/notification?auth_token=bar`?

**Comment by dahlbyk** [comment]
Oh.

> HipChat Endpoint (no trailing slashes):

**Pull request merged**